openmicroscopy/openmicroscopy
# thumbnail_default_size_96

See https://trello.com/c/abPmgATV/90-thumbnail-96-by-default

This changes the default size for webgateway thumbnails from 64 to 96, which
can then use the cached thumbnails from OMERO.

To test:
 - go to: ``` /webgateway/render_thumbnail/<imageId>/ ```
 - Should get a thumbnail that is 96 long on longest side.
 - ``` webgateway/render_birds_eye_view/<imageId>/ ``` should continue to give size 96 thumbnail as before (no change).
 - Dataset thumbnails no-longer have size specified (now use default size). Right-click on thumbnail and "Open image in new tab". Url should be ```webclient/render_thumbnail/<imageId>/``` with no '96' size.
 - Do the same for SPW thumbnails.
 - Check test is green at https://ci.openmicroscopy.org/job/OMERO-DEV-merge-integration-Python27/350/testReport/OmeroWeb.test.integration.test_plategrid/TestPlateGrid/test_get_plate_grid_metadata/
